<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClothSim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="s_transformGB">
      <property name="title">
       <string>Controls</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QComboBox" name="fixedComboBox">
         <item>
          <property name="text">
           <string>Top Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Top Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Top Middle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bottom Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bottom Right</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bottom Middle</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QDoubleSpinBox" name="windZSpin"/>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="massSpin"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="gravityLabel">
         <property name="text">
          <string>Gravity</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="windLabel">
         <property name="text">
          <string>Wind (x,y,z)</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="cHeightSpin">
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Add/Remove fixed m_particles</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="pDimLabel">
         <property name="text">
          <string>Particle Dimensions</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QDoubleSpinBox" name="windYSpin"/>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="gravitySpin">
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="value">
          <double>-9.810000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="addFixedButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="pWidthSpin">
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QDoubleSpinBox" name="windXSpin"/>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="pHeightSpin">
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QPushButton" name="togglePlay">
         <property name="text">
          <string>Toggle Play</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="clothDimLabel">
         <property name="text">
          <string>Cloth Dimensions</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="cWidthSpin">
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="createMeshButton">
         <property name="text">
          <string>Create Mesh/Reset</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="removeFixedButton">
         <property name="text">
          <string>Remove </string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="iterationsSpin"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1391</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
